models/calendar.yml

```diff
@@ -296,12 +296,19 @@
 
     - 9:00 PM:
         type: Talk
-        emoji: ✨
-        title: TBD
+        emoji: 💚
+        title: So Fast You'll Think It Broke: Content-Addressable Disk Caching in Pure JS
```

Comment:
I think the colon in `Broke:` is causing a YAML parse error here. Would you please try surrounding the title field in quotes?